src/main/java/net/sf/jabref/logic/importer/fileformat/MedlineImporter.java

```diff
@@ -80,7 +86,7 @@
  * check here for details on the format
  * https://www.nlm.nih.gov/bsd/licensee/elements_descriptions.html
  */
-public class MedlineImporter extends Importer {
+public class MedlineImporter extends Importer implements Parser {
```

Comment:
I did not quite follow the new developments at the importers -> why does this Importer need to implement Parser, while the other Importers do not. Or do all importers also do that?